README.md

```diff
@@ -11,10 +11,28 @@ $ npm install openwhisk
 
 ## usage
 
+Instantiate openwhisk with specific options:
+
 ```
-var openwhisk = require('openwhisk');
-var options = {apihost: 'openwhisk.ng.bluemix.net', api_key: '...'};
-var ow = openwhisk(options);
+function main(params) {
+	// require the openwhisk npm package
+	var openwhisk = require('openwhisk');
+
+	// read apihost, auth, and namespace from params
+	var apiHost = params.apiHost; # e.g. openwhisk.ng.bluemix.net
+	var namespace = params.namespace;
+	var auth = params.auth; 
+
+	// generate api_key from auth
+	var base64Auth = new Buffer(auth).toString('base64');
+	var apiKey = "Basic " + base64Auth;
```

Comment:
Did this example work for you? The user does not have to pass in the base64 string, the library handles that. It should be the raw auth string (user:pass).